.github/workflows/static-test.yml

```diff
@@ -24,6 +24,8 @@ jobs:
         # Note: ${{ github.base_ref }} is empty when not in a PR
         if [ -n "${{ github.base_ref }}" ]; then
           git fetch origin ${{ github.base_ref }}:${{ github.base_ref }} --no-tags
+        else
+          git config diff.renameLimit 16384
```

Comment:
I decided to use the next power of two after the quoted number of 9771 to have some time before we need to increase this value more.